openslides_backend/action/actions/poll/functions.py

```diff
@@ -0,0 +1,21 @@
+from ....permissions.permission_helper import has_perm
+from ....permissions.permissions import Permission, Permissions
+from ....services.datastore.interface import DatastoreService
+from ....shared.exceptions import MissingPermission
+from ....shared.patterns import KEYSEPARATOR
+
+
+def check_poll_or_option_perms(
+    content_object_id: str,
+    datastore: DatastoreService,
+    user_id: int,
+    meeting_id: int,
+) -> None:
+    if content_object_id.startswith("motion" + KEYSEPARATOR):
+        perm: Permission = Permissions.Motion.CAN_MANAGE_POLLS
+    elif content_object_id.startswith("assignment" + KEYSEPARATOR):
+        perm = Permissions.Assignment.CAN_MANAGE
+    else:
```

Comment:
I don't think it would be a good idea to do this in this PR, it has nothing to do with this feature and I would have to go through and change all of the tests that would fail as a result of that

Reply:
Created https://github.com/OpenSlides/openslides-backend/issues/2513 for this.